### ISSUE TITLE
One-hot-encode the country of origin

### ISSUE THREAD

**Issue by jwestw** [ThomasHSimm/Pesticide#17]
Could also try [label binarizer](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.LabelBinarizer.html#sklearn.preprocessing.LabelBinarizer)
